service/src/intTest/resources/application.yml

```diff
@@ -16,4 +19,11 @@ mongodb:
   options: ${GP2GP_MONGO_OPTIONS:}
   autoIndexCreation: ${GP2GP_MONGO_AUTO_INDEX_CREATION:true}
   ttl: ${GP2GP_MONGO_TTL:P30D}
-  cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
\ No newline at end of file
+  cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
+        include: info, health, metrics, mappings
+
+gp2gp:
+  storage:
+    type: ${GP2GP_STORAGE_TYPE:LocalMock}
+    containerName: ${GP2GP_STORAGE_CONTAINER_NAME:for-nia-testing}
+    azureConnectionString: ${GP2GP_AZURE_STORAGE_CONNECTION_STRING:for-nia-testing}
```

Comment:
Maybe leave off the "for-nia-testing" defaults here? These just happen to be the name we picked for the storage / S3 buckets but are not needed for "LocalMock".

Reply:
Ok